documents/Design.md

```diff
@@ -0,0 +1,25 @@
+# Dataset Viewer Design
+
+Programming Language: Python 3  
+Coding Style: PEP8  
+Design Pattern: Model View Presenter (Passive View)
```

Comment:
Please reference mslice as being the major source of inspiration for this, particularly in the MVP direction